apps/native-component-list/webpack.config.js

```diff
@@ -22,6 +22,15 @@ const locations = {
   modules: absolutePath('../../node_modules'),
 };
 
+let appManifest;
+try {
+  const PWAManifest = require(absolutePath('./web/manifest.json'));
```

Comment:
The PWA manifest is different from the Expo manifest -- for consistency with iOS & Android I think we should just expose the Expo manifest from Constants.manifest